x-pack/plugins/security_solution/public/cases/components/case_view/index.tsx

```diff
@@ -145,6 +145,7 @@ export const CaseView = React.memo(
     }, [dispatch]);
 
     const refreshRef = useRef<CaseViewRefreshPropInterface>(null);
+    const newComponent = true;
```

Comment:
A reminder to not forget removing it.

Reply:
these components will go to 🔥  I am keeping them temporarily to solve possible conflicts with other changes easily.